app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetControllerTest.kt

```diff
@@ -404,14 +395,6 @@ class BloodPressureEntrySheetControllerTest {
       onNext(SaveClicked)
     }
 
-    val newUserUuid = UUID.fromString("a726d885-b12d-40b7-9fe1-5391f3fc0f88")
```

Comment:
This was intentionally done to verify that if a different user than the one who records a BP edits it, the BP measurement is updated with the new user and facility ID.

Is this being tested elsewhere?

Reply:
We might want to add a different test for it so that this behaviour is tested explicitly rather than implicitly.

Reply:
I had an understanding of it when I removed the test. The test was incorrect because, it was not simulating the correct scenario. What we wanted was - what happens a different user updates the BP. What the test was doing - the user of the device has changed, now let's see if the BP entry has the updated user ID and the facility ID. Will add a test to cover this case.

Reply:
Added a test for this.